pkg/sql/vars.go

```diff
@@ -847,8 +847,9 @@ var varGen = map[string]sessionVar{
 					// TODO(knz): if/when we want to support this, we'll need to change
 					// the interface between GetStringVal() and Set() to take string
 					// arrays instead of a single string.
-					return "", unimplemented.Newf("schema names containing commas in search_path",
-						"schema name %q not supported in search_path", s)
+					return "", unimplemented.NewWithIssuef(53971,
+						`schema name %q has commas so is not supported in search_path.
+Did you mean to omit quotes? SET search_path = %s`, s, s)
```

Comment:
Might be nicer to make this a hint via `errors.WithHint`.

Reply:
I thought about that, but the unimplemented error type already uses a hint... I guess I could use two, or just not make it an unimplemented error, but this way we get telemetry on this hilarious situation.

Reply:
Hints are made to be chained / combined. We do this in other places too.

Reply:
Coming back to this several months later - thanks for the suggestion! Done.